Formula/parquet-cli.rb

```diff
@@ -27,6 +27,13 @@ class ParquetCli < Formula
     sha256 "5caf572cb0df5ce9d6893609de82d2369b42c3c81c611847b6f921d912040118"
   end
 
+  # Patches snappy to 1.1.8.4 for MacOS arm64 support, won't be needed in >= 1.13.0
+  # See https://issues.apache.org/jira/browse/PARQUET-2025
```

Comment:
Why not this commit: https://github.com/apache/parquet-mr/commit/095c78fec3378189296d38fede1255b0a4d05fd4 ?

Reply:
We can do this oneC do you want me to do the change and fixup, or in a new commit?

Reply:
Preferably it should all be one commit.

Reply:
Any updates on this @shanielh ?

Reply:
Hey @SMillerDev , if I create a new PR that uses https://github.com/apache/parquet-mr/commit/095c78fec3378189296d38fede1255b0a4d05fd4 for the patch, would it be alright to ping you for approval? :)

Reply:
Hi guys, sorry for the delay, I changed the fix to use that commit.